src/main/java/ruslan/shastkiv/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,30 @@
 package ruslan.shastkiv.bookstore.dto;
 
+import jakarta.validation.constraints.DecimalMin;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import ruslan.shastkiv.bookstore.validation.image.CoverImage;
+import ruslan.shastkiv.bookstore.validation.isbn.Isbn;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull(message = "Title cannot be null")
```

Comment:
use `@NotBlank` for String

```suggestion
    @NotBlank(message = "Title cannot be null")
```